libscript/src/script-module.cpp

```diff
@@ -1265,8 +1264,24 @@ bool MCScriptWriteInterfaceOfModule(MCScriptModuleRef self, MCStreamRef stream)
                     }
                     break;
                     case kMCScriptTypeKindRecord:
-                        // TODO - Records not yet supported
-                        break;
+                    {
+                        auto t_record_type =
```

Comment:
I think changing this `auto` to `MCScriptRecordType *` is the only change needed to get this merged.